appshell/cef_main_window.cpp

```diff
@@ -108,10 +144,14 @@ BOOL cef_main_window::Create()
     int top    = CW_USEDEFAULT;
     int width  = CW_USEDEFAULT;
     int height = CW_USEDEFAULT;
+
     int showCmd = SW_SHOW;
 
     LoadWindowRestoreRect(left, top, width, height, showCmd);
 
+    // make sure the window is visible 
+    EnsureWindowRectVisibility(left, top, width, height);
```

Comment:
also, I tested stretching the app across to monitors to ensure that would work. The origin https://github.com/adobe/brackets-shell/pull/144 implementation would constrain it to a single monitor.